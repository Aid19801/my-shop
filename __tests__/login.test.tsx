import { render, screen } from "@testing-library/react";
import Login from "../pages/login";

describe("LoginPage", () => {
  it("should render the page without exploding", () => {
    render(<Login />);
    const heading = screen.getByText("Login");
    expect(heading).toBeInTheDocument();
  });
  it("should match snapshot", () => {
    const { container } = render(<Login />);
    expect(container).toMatchSnapshot();
  });
});
