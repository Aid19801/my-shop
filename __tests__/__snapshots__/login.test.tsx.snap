// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage should match snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <h1
        class="title"
      >
        Login
      </h1>
      <form
        class="form"
        data-testid="login-form"
      >
        <input
          name="username"
          placeholder="username"
          type="text"
        />
        <input
          name="password"
          type="password"
        />
        <button
          type="submit"
        >
          Submit
        </button>
      </form>
    </main>
  </div>
</div>
`;
